docset/winserver2022-ps/netsecurity/Get-NetFirewallHyperVVMCreator.md

```diff
@@ -0,0 +1,125 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: NetFirewallHyperVVMCreator.cmdletDefinition.cdxml-help.xml
+Module Name: NetSecurity
+ms.date: 8/25/2023
+online version: https://docs.microsoft.com/powershell/module/netsecurity/get-netfirewallhypervvmcreator?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-NetFirewallHyperVVMCreator
+---
+
+# Get-NetFirewallHyperVVMCreator
+
+## SYNOPSIS
+Retrieves Hyper-V firewall VM Creators from the target computer.
+
+## SYNTAX
+
+### GetAll (Default)
+```
+Get-NetFirewallHyperVVMCreator [-All] [-CimSession <CimSession[]>] [-ThrottleLimit <int>] [-AsJob]  [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-NetFirewallHyperVVMCreator** cmdlet returns the instances of the Hyper-V firewall VM creators on the system.
+
+A Hyper-V firewall rule can be scoped to apply only to Hyper-V firewall ports created by specific Hyper-V VM creators. This cmdlet is used to query the list of creators currently on the system.
+
+A rule can scoped to a specific VM creator even if it is not yet on the system. This can be useful when configuring policy when a particular application may be installed at an unknown point in time.

```

Comment:
```suggestion
A rule can be scoped to a specific VM creator even if it is not yet on the system. This can be useful when configuring policy when a particular application may be installed at an unknown point in time.

```